pkg/model/issuerdiscovery.go

```diff
@@ -102,7 +102,7 @@ func (b *IssuerDiscoveryModelBuilder) Build(c *fi.ModelBuilderContext) error {
 
 func buildDiscoveryJSON(issuerURL string) ([]byte, error) {
 	d := oidcDiscovery{
-		Issuer:                fmt.Sprintf("%v/", issuerURL),
+		Issuer:                fmt.Sprintf("%v", issuerURL),
```

Comment:
```suggestion
		Issuer:                issuerURL,
```

Reply:
Let's leave it for a different PR, the cherry-pick just merged.

Reply:
Too late... :)